src/main/java/logIn/exceptions/UserException.java

```diff
@@ -0,0 +1,14 @@
+package logIn.exceptions;
+
```

Comment:
I'm not sure if exceptions should have its own package like this, as it implies it is a layer by itself.